modules/claims_api/spec/requests/v2/veterans/claims_request_spec.rb

```diff
@@ -612,6 +612,75 @@
             end
           end
         end
+
+        context 'it has an errored status' do
+          let(:bgs_claim) do
+            {
+              benefit_claim_details_dto: {
+                benefit_claim_id: '111111111',
+                claim_status_type: 'value from BGS',
+                bnft_claim_lc_status: [{
+                  phas_chngd_dt: DateTime.now,
+                  phase_type: 'Errored'
+                }, {
+                  phas_chngd_dt: DateTime.now - 1.day,
+                  phase_type: 'Initial Review'
+                }]
+              }
+            }
+          end
+
+          it "returns a claim with the 'claimId' and 'lighthouseId' set" do
+            with_okta_user(scopes) do |auth_header|
+              VCR.use_cassette('bgs/tracked_items/find_tracked_items') do
+                expect_any_instance_of(BGS::EbenefitsBenefitClaimsStatus)
+                  .to receive(:find_benefit_claim_details_by_benefit_claim_id).and_return(bgs_claim)
+                expect(ClaimsApi::AutoEstablishedClaim)
+                  .to receive(:get_by_id_or_evss_id).and_return(nil)
+
+                get claim_by_id_path, headers: auth_header
+
+                json_response = JSON.parse(response.body)
+                expect(response.status).to eq(200)
+                expect(json_response).to be_an_instance_of(Hash)
+                expect(json_response['claimType']).to eq('value from BGS')
+                expect(json_response['status']).to eq('ERRORED')
+              end
+            end
+          end
```

Comment:
This spec isn't necessary.
We're not going to get back claims from BGS that are errored.
The only errored claims will be Lighthouse claims.

Reply:
Then do I need [this line](https://github.com/department-of-veterans-affairs/vets-api/pull/10551/files#diff-ff0646b1c8016a6782bf1a5659159f1a2a9edf6cbd91b0a36a43215132deba0aR18)?

Reply:
Yeah, you'll still need that line.
The naming of that mapper class is unfortunate.  Because it doesn't _only_ handle mapping from a BGS status to a status that we control.  It also has to handle mapping our "pending" status to "PENDING" and our "errored" status to "ERRORED".